Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -96,6 +98,14 @@ var DrawerLayoutAndroid = React.createClass({
      */
     drawerWidth: ReactPropTypes.number,
     /**
+     * Specifies the lock mode of the drawer.
```

Comment:
What is a lock mode? Can you add documentation understandable for someone who doesn't know anything about the Android Drawer?